src/main/java/com/google/cloudassets/discovery/Main.java

```diff
@@ -24,9 +23,12 @@
     private static final String SPANNER_INSTANCE_ID = "spanner1";
     private static final String SPANNER_DATABASE_ID = "db1";
 
-    private static final String GET_PROJECTS_LIST_QUERY = "SELECT workspaceId, projectId "
-                                                        + "FROM Workspace_Project_Table "
-                                                        + "WHERE isActive = True";
+    private static final String GET_PROJECTS_LIST_QUERY = "SELECT s.workspaceId, p.projectId, "
+                                                        + "s.serviceAccountEmail\n"
+                                                        + "FROM Workspace_Project_Table as p inner join "
+                                                        + "Workspace_Service_Account_Table as s\n"
+                                                        + "on p.workspaceId = s.workspaceId\n"
```

Comment:
I changed the query formatting in the other active PR so upon approval it will merge in the master branch:
https://github.com/googleinterns/step265-2020/pull/17/commits/4a8503d51234654eab6e0e9f911921c5083f3e0f